src/movie.c

```diff
@@ -319,131 +319,156 @@ GMT_LOCAL int movie_parse_x_option (struct GMT_CTRL *GMT, struct MOVIE_CTRL *Ctr
+	GMT_Usage (API, 3, "+t Provide a C-format statement to be used with the item selected [none].");
+	GMT_Usage (API, 3, "+w Specify indicator size [5%% of max canvas dimension for circles, 60%% for axes].");
+	GMT_Usage (API, 1, "\n-Q[s]");
+	GMT_Usage (API, -2, "Debugging: Leave all intermediate files and directories behind for inspection. "
+		"Append s to only create the work scripts but none will be executed (except for background script).");
+	GMT_Usage (API, 1, "\n-Sb<background>");
+	GMT_Usage (API, -2, "Append name of background GMT modern script that may pre-compute "
+		"files needed by <mainscript> and/or build a static background plot layer. "
+		"If a plot is generated then the script must be in GMT modern mode. "
+		"Alternatively, give PostScript file of correct canvas size that will be the background.");
+	GMT_Usage (API, 1, "\n-Sb<foreground>");
+	GMT_Usage (API, -2, "Append name of foreground GMT modern mode script which will "
+		"build a static foreground plot overlay appended to all frames. "
+		"Alternatively, give PostScript file of correct canvas size that will be the foreground.");
 	GMT_Option (API, "V");
-	GMT_Message (API, GMT_TIME_NONE, "\t-W Give <workdir> where temporary files will be built [<workdir> = <prefix> set by -N].\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   If <workdir> is not given we create one in the system temp directory named <prefix> (from -N).\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t-Z Erase directory <prefix> after converting to movie [leave directory with PNGs alone].\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   Append s to also delete all input scripts (mainscript and any files via -E, -I, -S)\n");
+	GMT_Usage (API, 1, "\n-W[<workdir>]");
+	GMT_Usage (API, -2, "Give <workdir> where temporary files will be built [Default <workdir> = <prefix> set by -N]. "
+		"If <workdir> is not given we create one in the system temp directory named <prefix> (from -N).");
+	GMT_Usage (API, 1, "\n-Z[s]");
+	GMT_Usage (API, -2, "Erase directory <prefix> after converting to movie [leave directory with PNGs alone]. "
+		"Append s to also delete all input scripts (mainscript and any files via -E, -I, -S).");
 	GMT_Option (API, "f");
 	/* Number of threads (re-purposed from -x in GMT_Option since this local option is always available and we are not using OpenMP) */
-	GMT_Message (API, GMT_TIME_NONE, "\t-x Limit the number of cores used in frame generation [Default uses all cores = %d].\n", API->n_cores);
-	GMT_Message (API, GMT_TIME_NONE, "\t   -x<n>  Select <n> cores (up to all available).\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   -x-<n> Select (all - <n>) cores (or at least 1).\n");
+	GMT_Usage (API, -2, "Limit the number of cores used in frame generation [Default uses all cores = %d]." , API->n_cores);
+	GMT_Usage (API, 3, "x<n>  Select <n> cores (up to all available).");
+	GMT_Usage (API, 3, "-x-<n> Select (all - <n>) cores (or at least 1).");
```

Comment:
these three lines are not formatted well

Reply:
Fixing this for both movie|batch as well as for the OpenMP version as well.
Right now -x says its args are optional.  So what does -x mean? Per the code it means use all the cores, which it says is default... So perhaps change this to be a required argument, since we cannot start saying -x is required to use many cores.  OK?